Kaguya/Kaguya/Discord/Commands/Configuration/Antiraid.cs

```diff
@@ -253,6 +372,41 @@ public async Task AntiraidToggleCommand()
                                       $"Config:\n{GetAntiraidConfigString(curConfig)}", KaguyaColors.LightYellow);
         }
 
+        [Command("-setmsg")]
+        [Summary("Sets and enables the antiraid direct messages (DMs) for this server. Antiraid DMs " +
+                 "are messages that get sent to each individually actioned user by the antiraid service.\n\n" +
+                 "This is useful if you want to be sure legitimate users can be notified of why they " +
+                 "were actioned from your server (i.e. " +
+                 "legitimate user joins your server while a separate raid is in-progress). This message can " +
+                 "be whatever you want these users to read after being actioned. It's not a bad idea to include a " +
+                 "link to a ban appeal or a permanent invite link, depending on how you action raids in your server.\n\n" +
```

Comment:
Should instead be:

If there is a false-positive (i.e. legitimate user joins the server while a raid is in progress), they can be sent a custom DM with perhaps an invite link or ban appeal form.

Reply:
Should also add: 

"Antiraid DMs that violate Discord's Terms of Service (LINK) will result in a permanent blacklist from Kaguya."